### ISSUE TITLE
No LICENSE

### ISSUE THREAD

**Issue by haliphax** [unfug-deluxe/lightbarmain-x84#1]
You should include a LICENSE file for your project. ;)

**Comment by unfug-deluxe** [comment]
Hi haliphax

will do, but which license is the most common with x/84? Will adapt to that.

**Comment by haliphax** [comment]
x/84 itself uses the MIT license, but you can use any license you'd like.

**Comment by unfug-deluxe** [comment]
Ah, okay. So it's MIT license for me. Thanks for the feedback :)

**Comment by haliphax** [comment]
Just (hopefully) being a good open-source citizen, my friend. Hopefully I didn't come off as negative. Cheers!

**Comment by unfug-deluxe** [comment]
Nope, u didn't - am an old C coder long gone into *NIX administration. Am new to all this github and community stuff. Happy to participate - noob or not ;)
